### PR TITLE
Updates based on recent content review

### PR DESCRIPTION
**Pull request by kylekirkby**
Updates based on recent content review

@pcolmer

### CODE REVIEW

File: _pages/code_of_conduct.md [hunk 1]
```diff
@@ -17,12 +17,12 @@ Linaro Ltd, as host for Linaro Connect, is dedicated to providing a harassment-f
...
 
 If a participant engages in harassing behavior, the conference organizers may take any action they deem appropriate, ranging from issuance of a warning to the offending individual to expulsion from the conference with no refund, depending on the circumstances. Linaro reserves the right to exclude any participant found to be engaging in harassing behavior from participating in any further Linaro events, trainings or other activities.
 
-If you are being harassed, notice that someone else is being harassed, or have any other concerns relating to harassment, please contact a member of conference staff immediately. Conference staff can be identified by t-shirts/staff badges onsite; and an organizer can be found at the event registration counter at any time. You are also encouraged to contact Diane Cheshire, Group HR Director at diane (dot) cheshire (at) linaro (dot) org.
```
Comment (pcolmer) [inline _pages/code_of_conduct.md]: Aw, shucks. I thought we were going to get special t-shirts or badges :)

File: _pages/code_of_conduct.md [hunk 2]
```diff
@@ -17,12 +17,12 @@ Linaro Ltd, as host for Linaro Connect, is dedicated to providing a harassment-f
...
+If you are being harassed, notice that someone else is being harassed, or have any other concerns relating to harassment, please contact a member of conference staff immediately. An organizer can be found at the event registration counter at any time. You are also encouraged to contact Diane Cheshire, Group HR Director at diane (dot) cheshire (at) linaro (dot) org.
```
Comment (pcolmer) [inline _pages/code_of_conduct.md]: I think Kristine needs to review this wording. There isn't an organizer at the registration desk **at any time**.

Comment (kylekirkby) [reply _pages/code_of_conduct.md]: @EbbaSimpson @kristinedill - could you take a look at the wording used here?

**Comment by kristinedill** [comment]
Hi @kylekirkby just stumbled on this request - not sure how I missed it. 
I tried to update the text in the code of conduct, but it's asking me for a new background image before I can save on Netlify. Is there a theme to all the pages or should I just add a random flikr image? 

Thanks, 
Kristine

**Comment by kylekirkby** [comment]
> Hi @kylekirkby just stumbled on this request - not sure how I missed it.
> I tried to update the text in the code of conduct, but it's asking me for a new background image before I can save on Netlify. Is there a theme to all the pages or should I just add a random flikr image?
> 
> Thanks,
> Kristine

Hi @kristinedill ,

That shouldn't be the case! I'll push an update to the NetlifyCMS config now :) 

Thanks,

Kyle

**Comment by kylekirkby** [comment]
Fixed in this PR: https://github.com/Linaro/Connect/pull/381 :)

**Pull request merged**